src/components/shared/ExternalLink/types.ts

```diff
@@ -0,0 +1,5 @@
+import { type HTMLProps } from 'react';
+
+export interface ExternalLinkProps extends HTMLProps<HTMLAnchorElement> {
+  hasHover?: boolean;
+}
```

Comment:
This is perfectly fine but I guess for a more strict type safety, we can import `AnchorHTMLAttributes` here:

```js
import { AnchorHTMLAttributes } from 'react';

export interface ExternalLinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
  hasHover?: boolean;
}

```